contents/docs/toolbar/heatmaps.mdx

```diff
@@ -31,33 +37,45 @@ There are three kinds of heatmaps in the toolbar:
 
 ## Heatmaps
 
-The heatmap shows where on the page users are clicking, dead clicking, rageclicking, or moving their mouse. Unlike the clickmap, the user doesn't need to click on a clickable element, so you can see where users are trying to click but can't. 
+The heatmap shows where on the page users are clicking, dead clicking, rageclicking, or moving their mouse. Unlike the clickmap, the user doesn't need to click on a clickable element, so you can see where users are trying to click.
 
 There are multiple options to configure your heatmap.
 
 ![Heatmap settings](https://res.cloudinary.com/dmukukwp6/image/upload/v1716592885/posthog.com/contents/docs/toolbar/settings.png)
 
 ### Aggregation
 
-The heatmap can show a count of events or of unique users. The default is to show the count of events.
+The heatmap can show a count of events, or of unique users. The default is to show the count of events.
+
+The unique users count is based on the user's distinct ID, so if a user is using multiple devices or browsers, they are counted as multiple users. 
+
+This is so that the heatmap can be much faster to load.
```

Comment:
```suggestion
The unique users count is based on the user's distinct ID, so if a user is using multiple devices or browsers, they are counted as multiple users. This is so that the heatmap can be much faster to load.
```